l10n_ch_qr_no_amount/report/swissqr_no_amount_report.xml

```diff
@@ -39,17 +29,29 @@
             <attribute name="t-value">str()</attribute>
         </xpath>
         <xpath
-            expr="//div[@id='receipt_amount_zone']//span[@t-esc='formated_amount']"
-            position="replace"
+            expr="//div[@id='receipt_amount_zone']//span[@t-out='formated_amount']"
+            position="attributes"
+        >
+            <attribute name="t-if">False</attribute>
```

Comment:
@sonhd91 do you have any particular reasons behind this refactor? Otherwise we should keep the original code